### PR TITLE
the mobile navbar feature

### PR DESCRIPTION
**Pull request by yusufobr**
In this PR, the bellow features have been added : 

- Added Html & CSS for the Mobile menu.
- Added JavaScript animation to open and close the menu.
- Update the README file for the new feature.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -387,6 +395,54 @@ form button {
...
+    list-style: none;
+  }
+
+  .a-mobile {
+    font-size: 32px;
+    font-weight: 600;
+    text-decoration: none;
+    color: white;
+  }
+}
```
Comment (omarsalem7) [inline style.css]: - [ ] Kindly consider adding a [backdrop filter](https://developer.mozilla.org/en-US/docs/Web/CSS/backdrop-filter) for the mobile menu to match your design with the Figma design 

| Your design  | Figma design |
| ------------- | ------------- |
| image  | image |

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @yusufobr 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Mobile menu opens and closes properly  ✔️
- The Pull Request has a proper title and description. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (yusufobr) [reply style.css]: Hello @omarsalem7,
okay, thank you for your review.
the required change is taken into consideration.

**Review by omarsalem7 — APPROVED** [review]
Hi @yusufobr 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @AnsarIbrahim 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛑🛑
yPkfDFf

This repository belongs to @yusufobr, And the project has already been approved. Please collaborate and request a review from the @AnsarIbrahim project as a new one. 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by nicupop729** [comment]
@yusufobr & @AnsarIbrahim please find [here the Work and submission mode](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode) for this milestone.

Kindly make sure to collaborate on the repository of @AnsarIbrahim too when implementing the project requirements of this milestone.

**Pull request merged**